pjmedia/src/pjmedia-videodev/util.c

```diff
@@ -164,8 +164,8 @@ pjmedia_vid_dev_conv_create_converter(pjmedia_vid_dev_conv *conv,
     
     conv->wxh = conv->dst_size.w * conv->dst_size.h;
     conv->src_frame_size = dst_size.w * dst_size.h * vfi->bpp / 8;
-    conv->conv_frame_size = conv->rot_size.w * conv->rot_size.h *
-			    vfi->bpp / 8;
+    conv->conv_frame_size = conv->rot_size.w * conv->rot_size.h;
+    conv->conv_frame_size *= vfi->bpp / 8;
```

Comment:
this change should not be reverted, note that for IYUV/I420, bpp==12, so conv_frame_size will be multiplied to 1 instead of 1.5.